mipengine/node/tasks/merge_tables.py

```diff
@@ -46,7 +46,7 @@ def create_merge_table(context_id: str, command_id: str, table_names: List[str])
     validate_tables_can_be_merged(table_names)
     schema = common_actions.get_table_schema(table_names[0])
     merge_table_name = create_table_name(
-        "merge", command_id, context_id, config.node.identifier
+        "merge", command_id, context_id, config.identifier
     )
```

Comment:
again...
1. define enums, do not put arbitrary strings like "merge" in there
2. why create_table_name(..) function need to be inside the db interface module?

Reply:
Not related to this PR, please make a separate ticket.